components/Storyblok/SbEventBanner.tsx

```diff
@@ -68,6 +70,7 @@ export const SbEventBanner = ({
     <EventBanner
       {...storyblokEditable(blok)}
       id={id}
+      superhead={superhead || 'Event'}
```

Comment:
Passing a fallback superhead here. I found out that if we pass that in the variable higher up, sometimes Storyblok empty string can override the default.